build.sbt

```diff
@@ -29,15 +30,15 @@ val reactiveStreamsV = "1.0.2"
 
 val specs2V = "4.5.1"
 
-val kindProjectorV = "0.10.0"
-val betterMonadicForV = "0.3.0"
+val kindProjectorV = "0.9.9"
+val betterMonadicForV = "0.3.0-M4"
```

Comment:
These downgrades avoid a cross-versioning.  We'll reupgrade on the way back to RC1.